options/posix/include/search.h

```diff
@@ -15,6 +15,25 @@ typedef enum {
   leaf
 } VISIT;
 
+typedef enum {
```

Comment:
Same for this here, we can include the glibc header in here if the glibc option is enabled.